smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/swiftmodules/SmithyFormURLTypes.kt

```diff
@@ -9,12 +9,17 @@ import software.amazon.smithy.swift.codegen.SwiftDeclaration
 import software.amazon.smithy.swift.codegen.SwiftDependency
 
 object SmithyFormURLTypes {
-    val Writer = runtimeSymbol("Writer", SwiftDeclaration.CLASS)
+    val Writer = runtimeSymbol("Writer", SwiftDeclaration.CLASS, listOf(SmithyReadWriteTypes.SmithyWriter))
```

Comment:
`FormURLWriter` is made SPI, and imports the SmithyWriter protocol along with itself.